### ISSUE TITLE
Implement iHAMMOC experiment

### ISSUE THREAD

**Issue by gold2718** [NorESMhub/CAM#127]
### What is the feature/what would you like to discuss?

iHAMMOC: Merge the `feature-hamocc-vsls` branch into noresm2_3_develop.
- Create (if necessary) compset or user_mods for demonstration case
- Create regression test using the case above
- Create final tag for feature-hamocc-vsls branch
- After merge, rerun regression test to ensure answer preservation
- Create new tag in cam_cesm2_1_rel_05-Nor

### Will this change (regression test) answers?

If a regression test is included that tests the features specific to the `feature-hamocc-vsls` branch, it may change answers for that test.

### Will you be implementing this enhancement yourself?

Yes

**Comment by TomasTorsvik** [comment]
The `feature-hamocc-vsls` branch is 3 years old. As far as I can see, an updated version of vsls was merged into `master` with the commit 
https://github.com/NorESMhub/BLOM/commit/d374932bf2392404475f1ac1d991ef61c5f85d8a

**Comment by oyvindseland** [comment]
Are there feature still missing in CAM for bromoform in 2.3? 
If so I guess this is a matter for 2.3.1. @DirkOlivie

**Comment by TomasTorsvik** [comment]
At the moment I don't think this is an iHAMOCC issue. Is this issue related to issue #99 ?

**Comment by oyvindseland** [comment]
Yes. Although this specific issue is under CAM so should be independent on whether it  works in iHAMOCC or NorESM.
